### DISCUSSION TITLE
How to move window to next or previous space?

### DISCUSSION THREAD

**Discussion by szfkamil** [asmvik/yabai#2195] [General]
You can move spaces to the left or right, but there's no intuitive way of moving windows to a space to the right or left. Has anyone done this using rules and signals? I'm personally not very familiar with them, and I think someone must have hacked something that works like this already.

**Comment by asmvik** [comment]
yabai -m window --space next ?

**Reply by szfkamil** [reply]
Yes… It's not contained in the wiki, that's why I thought there is no such command.

**Reply by asmvik** [reply]
The wiki is just a sample of some commands. Use https://github.com/koekeishiya/yabai/blob/master/doc/yabai.asciidoc for actual documentation.

**Reply by szfkamil** [reply]
Thank you. I would just like to point out that what the wiki states at the beginning might be a bit misleading:

> This wiki page aims to explain every command available in detail.

This is why I had no awareness of the command and always looked to the wiki first. I'll use the `man` page instead from now on!

**Reply by asmvik** [reply]
Edited now (thought I had already done that..).
